Dockerfile

```diff
@@ -1,9 +1,8 @@
 ARG NODE_VERSION=18
-FROM node:${NODE_VERSION}-alpine3.18
+FROM node:${NODE_VERSION}-alpine3.20
 
 LABEL org.opencontainers.image.source="https://github.com/Countingup/docker-node"
 
-RUN apk add --no-cache --update --upgrade git openssh-client make bash lftp coreutils zip jq busybox \
+RUN apk add --no-cache --update --upgrade git openssh-client make bash lftp coreutils zip jq \
     curl groff less python3 py-pip && \
-    pip3 install --no-cache-dir --upgrade pip && \
```

Comment:
Why are we no longer upgrading pip?

Reply:
I think this is a holdover from an ancient alpine version with outdated python.

But with alpine3.20 this command refuses to run as pip is supposed to be managed by apk -- I could force it with the flag added below, but there's literally no value in this step, so I removed instead.